src/TokenParser/AbstractTokenParser.php

```diff
@@ -29,4 +33,26 @@ public function setParser(Parser $parser): void
     {
         $this->parser = $parser;
     }
+
+    protected function parseAssignmentExpression()
```

Comment:
this should have a return type (and probably some phpdoc explaining what it parsed in more details)

Reply:
Done